src/package_manager/pip.ts

```diff
@@ -57,10 +57,17 @@ const pip3Packages: string[] = [
 	"pytest-repeat",
 	"pytest-rerunfailures",
 	"pytest-runner",
-	"setuptools<60.0",
+	"setuptools",
```

Comment:
What is this change meant to test?

Reply:
We need newer setuptools due to [distutils removal since python3.12](https://docs.python.org/dev/whatsnew/3.12.html) somehow.

Reply:
We should probably instead pin Python to version 3.10 on macOS and 3.8 on Windows, since those are the currently-targeted Python versions according to REP 2000: https://www.ros.org/reps/rep-2000.html#rolling-ridley-june-2020-ongoing

Reply:
Bumping Python to 3.8 on Windows in #677

Reply:
Thank you for letting me know that. I created [a new PR](#678) to pin pyhton3.10 for osx.